mlir-sycl/test/Transforms/inliner.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: sycl-mlir-opt -always-inline -split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
Do you really need `-split-input-file`? looks like you are not using it.

Reply:
Sure but does not hurt and later we will add more tests to this file.